src/libs/utils.ts

```diff
@@ -99,6 +99,10 @@ export const differenceFromNowInSeconds = (jsonDateString) => {
   return differenceInSeconds(parseJSON(jsonDateString), Date.now());
 };
 
+export const differenceFromDatesInSeconds = (jsonDateStringStart, jsonDateStringEnd) => {
```

Comment:
Can you add unit test coverage for this (similar to what exists for `differenceFromNowInSeconds`)?